src/orquestra/sdk/_ray/_dag.py

```diff
@@ -187,23 +187,14 @@ class RayParams:
 class RayRuntime(RuntimeInterface):
     def __init__(
         self,
-        client: RayClient,
         config: RuntimeConfiguration,
         project_dir: Path,
+        client=None,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Can you add type annotations for `client`?